src/lib/CookieDecoder.ts

```diff
@@ -31,6 +33,16 @@ export class CookieDecoder {
         return wallets;
     }
 
+    // This method is taken from @nimiq/core's Nimiq.Address
+    public static toUserFriendlyAddress(serializedAddress: Uint8Array, withSpaces = true): string {
+        const base32 = this._toBase32(serializedAddress);
```

Comment:
I would move this to e.g. `/lib/AddressHelper.ts`. (As long as it is not possible to import such things from a core package.)